### PR TITLE
first commit on first break

### PR DESCRIPTION
**Pull request by khalilMoutik**
i finish class diagram

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@ node_modules/
 npm-debug.log
 yarn-error.log
 
+# Laravel 4 specific
```
Comment (salasou) [inline .gitignore]: remove dupplicated

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# clinique-virtuelles
\ No newline at end of file
+# clinique-virtuelle
```
Comment (salasou) [inline README.md]: add conf server

**Review by salasou — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (khalilMoutik) [reply README.md]: done

File: .gitignore (hunk 1 above)
Comment (khalilMoutik) [reply .gitignore]: done

**Pull request merged**